src/amp-story-player/amp-story-player-impl.js

```diff
@@ -177,6 +178,54 @@ export class AmpStoryPlayer {
     this.layoutCallback();
   }
 
+  /**
+   * Adds stories to the player, both adding new anchor tags to the player,
+   * and creating or assigning iframes to those that are close to the
+   * current playing story.
+   *
+   * @param {!Array<!{href: string, title: ?string}>} stories
```

Comment:
Now that I think about it, maybe we should also accept an optional `posterImage` property to set as the `data-poster-portrait-src` attribute on the anchor. This might be useful for showing a "preview" of the next story using the thumbnail image.